### PR TITLE
[DailyMail] Improve title and description extraction

### PR DESCRIPTION
**Pull request by TRox1972**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Wrap title in `unescapeHTML`

### CODE REVIEW

File: youtube_dl/extractor/dailymail.py [hunk 1]
```diff
@@ -26,7 +27,6 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, video_id)
         video_data = self._parse_json(self._search_regex(
             r"data-opts='({.+?})'", webpage, 'video data'), video_id)
-        title = video_data['title']
```
Comment (dstftw) [inline youtube_dl/extractor/dailymail.py]: Title should be extracted first. Also add a test.

**Comment by TRox1972** [comment]
@dstftw Does this look good?

File: youtube_dl/extractor/dailymail.py [hunk 2]
```diff
@@ -19,14 +20,17 @@ class DailyMailIE(InfoExtractor):
             'title': 'Turn any video into an impressionist masterpiece',
             'description': 'md5:88ddbcb504367987b2708bb38677c9d2',
         }
-    }
+    }, {
+        'url': 'www.dailymail.co.uk/video/tvshowbiz/video-1295863/The-Mountain-appears-sparkling-water-ad-Heavy-Bubbles.html',
+        'only_matching': True,
```
Comment (dstftw) [inline youtube_dl/extractor/dailymail.py]: It does not test the purpose of this change, i.e. title extraction improvement. It should replace the old test.

**Comment by TRox1972** [comment]
@dstftw Good?

**Comment by dstftw** [comment]
Cherry-picked in 111de00289d8c019764e79247568248f8a4b11f6, thanks.

**Pull request closed without merging**